src/main/resources/schemas/schema-form.json

```diff
@@ -34,6 +34,12 @@
       "default": 2000,
       "minimum": 0
     },
+    "releaseCache": {
```

Comment:
As seen with @ibahloul we should move this attribute just after `name` to clarify the meaning for the users